com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/Settings/FrameSettings.cs

```diff
@@ -816,7 +816,7 @@ internal static void Sanitize(ref FrameSettings sanitizedFrameSettings, Camera c
             //MSAA only supported in forward and when not using ray tracing.
             if (sanitizedFrameSettings.litShaderMode != LitShaderMode.Forward || pipelineSupportsRayTracing)
                 sanitizedFrameSettings.msaaMode = MSAAMode.None;
-            bool msaa = (sanitizedFrameSettings.msaaMode == MSAAMode.FromHDRPAsset && renderPipelineSettings.msaaSampleCount != MSAASamples.None) || sanitizedFrameSettings.msaaMode != MSAAMode.None;
+            bool msaa = sanitizedFrameSettings.msaaMode == MSAAMode.FromHDRPAsset ? renderPipelineSettings.msaaSampleCount != MSAASamples.None : sanitizedFrameSettings.msaaMode != MSAAMode.None;
```

Comment:
@JulienIgnace-Unity please review this one

Reply:
We did talk about this with Julien, but worth giving the thing a re read yeah